README.md

```diff
@@ -108,7 +108,7 @@ object Main {
     
     val myComponent = div("Hello World")
 
-    OutWatch.renderReplace("#app", myComponent).unsafeRunSync()
+    OutWatch.renderReplace[IO]("#app", myComponent).unsafeRunSync()
```

Comment:
Having to specify this type parameter here each time is a little annoying. It makes me wonder if the base functions on Outwatch could be wrapped into a trait where the `F` is specified, similar to `MonixOps[F]`.

Reply:
This works now.